www/include/configuration/configObject/service/listServiceByHost.php

```diff
@@ -72,29 +72,18 @@
     $status = $_GET["status"];
 }
 
-if (isset($_POST['Search'])) {
-    $centreon->historySearch[$url] = array();
-    $template = $_POST["template"];
-    $centreon->historySearch[$url]["template"] = $template;
-    $status = $_POST["status"];
-    $centreon->historySearch[$url]["status"] = $status;
-    $searchH = $_POST["searchH"];
-    $centreon->historySearch[$url]["searchH"] = $searchH;
-    $searchS = $_POST["searchS"];
-    $centreon->historySearch[$url]["searchS"] = $searchS;
-    isset($_POST["statusHostFilter"]) ? $hostStatus = 1 : $hostStatus = 0;
-    $centreon->historySearch[$url]["hostStatus"] = $hostStatus;
-} elseif (isset($_GET['Search'])) {
-    $centreon->historySearch[$url] = array();
-    $template = $_GET['template'];
+if (isset($_POST['search']) || isset($_GET['search'])) {
+    $data = isset($_POST['search']) ? $_POST : $_GET;
```

Comment:
I checked again this page and the XHR request : 
When you do a search, a POST request is sent with the data
The GET is only used to initialize the page.
So no need to keep the isset($_GET['search']), as it won't happen
Don't need to add a new variable $data neither. Use the $_POST[" xx "]  directly